examples/policies/soul-bound-policy/soul-bound-policy-v1.pact

```diff
@@ -0,0 +1,111 @@
+(namespace (read-msg 'ns))
+
+(module soul-bound-policy-v1 GOVERNANCE
+
+  (defconst ADMIN-KS:string "marmalade-v2.marmalade-contract-admin")
+
+  (defcap GOVERNANCE ()
+    (enforce-guard ADMIN-KS))
+
+  (implements kip.token-policy-v2)
+  (use kip.token-policy-v2 [token-info])
+  (use marmalade-v2.policy-manager)
+
+  (defschema mint-record 
+    account:string
+  )
+
+  (deftable records:{mint-record})
+
+  (defun key:string (tokenId:string account:string)
```

Comment:
I think the key here should only include `tokenId`. 
In this current setup it's possible to transfer the token to another account by burning on the originating account and minting again on the new account which is what we want to prevent in this example